downloads.html

```diff
@@ -5,12 +5,52 @@
 	<div class="right">
     <h1>Download</h1>
 
-    <p>2.4.1 is the latest release. The current stable version is 2.4.1.</p>
+    <p>2.5.0 is the latest release. The current stable version is 2.5.0.</p>
 
     <p>
     You can verify your download by following these <a href="https://www.apache.org/info/verification.html">procedures</a> and using these <a href="https://www.apache.org/dist/kafka/KEYS">KEYS</a>.
     </p>
 
+    <span id="2.5.0"></span>
+    <h3 class="download-version">2.5.0<a href="#2.5.0"><i class="fas fa-link " style="color:#053ce2"></i></a></h3>
+    <ul>
+        <li>
+            Released April 15, 2020
+        </li>
+        <li>
+            <a href="https://www.apache.org/dist/kafka/2.5.0/RELEASE_NOTES.html">Release Notes</a>
+        </li>
+        <li>
+            Source download: <a href="https://www.apache.org/dyn/closer.cgi?path=/kafka/2.5.0/kafka-2.5.0-src.tgz">kafka-2.5.0-src.tgz</a> (<a href="https://www.apache.org/dist/kafka/2.5.0/kafka-2.5.0-src.tgz.asc">asc</a>, <a href="https://www.apache.org/dist/kafka/2.5.0/kafka-2.5.0-src.tgz.sha512">sha512</a>)
+        </li>
+        <li>
+            Binary downloads:
+            <ul>
+                <li>Scala 2.12 &nbsp;- <a href="https://www.apache.org/dyn/closer.cgi?path=/kafka/2.5.0/kafka_2.12-2.5.0.tgz">kafka_2.12-2.5.0.tgz</a> (<a href="https://www.apache.org/dist/kafka/2.5.0/kafka_2.12-2.5.0.tgz.asc">asc</a>, <a href="https://www.apache.org/dist/kafka/2.5.0/kafka_2.12-2.5.0.tgz.sha512">sha512</a>)</li>
+                <li>Scala 2.13 &nbsp;- <a href="https://www.apache.org/dyn/closer.cgi?path=/kafka/2.5.0/kafka_2.13-2.5.0.tgz">kafka_2.13-2.5.0.tgz</a> (<a href="https://www.apache.org/dist/kafka/2.5.0/kafka_2.13-2.5.0.tgz.asc">asc</a>, <a href="https://www.apache.org/dist/kafka/2.5.0/kafka_2.13-2.5.0.tgz.sha512">sha512</a>)</li>
+            </ul>
+            We build for multiple versions of Scala. This only matters if you are using Scala and you want a version
+            built for the same Scala version you use. Otherwise any version should work (2.12 is recommended).
+        </li>
+    </ul>
+
+    <p>
+        Kafka 2.5.0 includes a number of significant new features. Here is a summary of some notable changes:
+    </p>
+
+    <ul>
+        <li>TLS 1.3 support (1.2 is now the default)</li>
+        <li>Co-groups for Kafka Streams</li>
+        <li>Incremental rebalance for Kafka Consumer</li>
+        <li>New metrics for better operational insight</li>
+        <li>Upgrade Zookeeper to 3.5.7</li>
+        <li>Deprecate support for Scala 2.11</li>
+    </ul>
+
+    <p>
+        For more information, please read the detailed <a href="https://www.apache.org/dist/kafka/2.5.0/RELEASE_NOTES.html">Release Notes</a>.
+    </p>
+
```

Comment:
These should have been added to the kafka repository in the `documentation/upgrade.html` file. 

See https://github.com/apache/kafka/pull/9118 that adds this section back into the correct location/repository, and see https://github.com/apache/kafka/pull/9117 for a similar fix for the 2.6.0 release. See https://github.com/apache/kafka/pull/9119 for a fix for the `trunk` branch.